src/bandwidthRtc.ts

```diff
@@ -88,12 +92,17 @@ class BandwidthRtc {
    * @param audioLevelChangeHandler handler that can be called when the audio level of the published stream changes (optional)
    * @param alias stream alias/tag that will be included in subscription events and billing records, should not be PII (optional)
    */
-  async publish(input?: MediaStreamConstraints | MediaStream, audioLevelChangeHandler?: AudioLevelChangeHandler, alias?: string): Promise<RtcStream> {
+  async publish(
+    input?: MediaStreamConstraints | MediaStream,
+    audioLevelChangeHandler?: AudioLevelChangeHandler,
+    alias?: string,
+    codecPreferences?: CodecPreferences
```

Comment:
We should include doc for this parameter. 

As an aside should we agree on a set of conventions for documenting functions? I found this but don't know enough to say whether it is the right way to do it: 
https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html

Reply:
Done. Yeah we should. I think I've been trying to follow that more or less, there's also https://tsdoc.org/ which I guess is still a WIP.